pkg/security/tests/process_test.go

```diff
@@ -434,6 +406,41 @@ func TestProcessContext(t *testing.T) {
 		})
 	})
 
+	t.Run("tty-raw", func(t *testing.T) {
```

Comment:
I'm not sure about the purpose of this test vs the previous one ? is it to test the kernel value vs the fallback to proc ?

Reply:
Yes exactly. The root cause was that tty tests were passing even though the constants were wrong, because of the fallback mechanism. This test ensure that the value we get from the kernel event is actually correct

Reply:
I think the "raw" still test the fallback, as the resolution is done before the DispatchEvent
https://github.com/DataDog/datadog-agent/blob/main/pkg/security/probe/probe.go#L504 ?

Reply:
I agree with you but from my experience running the testcases: when the constant is wrong, the previous test is passing and the new test is failing. If you want I can skip this new test (or split it in a separate PR)

Reply:
I move the tty-raw test to a separate PR (https://github.com/DataDog/datadog-agent/pull/10744/)